batcher/storage/initial_storage_ghostnet.mligo

```diff
@@ -23,39 +23,47 @@ let f(_:unit) = {
   ];
   valid_swaps = Map.literal [
     ("tzBTC/USDT", {
-        from = {
-          amount = 1n;
-          token = {
-            name = "tzBTC";
-            address = Some(("KT1XLyXAe5FWMHnoWa98xZqgDUyyRms2B3tG" : address));
-            decimals = 8;
-            standard = Some "FA1.2 token";
-          };
+        swap = {
+            from = {
+              amount = 1n;
+              token = {
+                name = "tzBTC";
+                address = Some(("KT1XLyXAe5FWMHnoWa98xZqgDUyyRms2B3tG" : address));
+                decimals = 8;
+                standard = Some "FA1.2 token";
```

Comment:
I think using something like
`type standard =
  | FA2
  | FA12
  | Other of string`

would be better.

Reply:
I'll add it as an enhancement issue

Reply:
https://github.com/marigold-dev/batcher/issues/258